package.json

```diff
@@ -15,11 +15,17 @@
   "devDependencies": {
     "chai": "^3.3.0",
     "mocha": "^2.3.3",
-    "tcp-port-used": "^0.1.2"
+    "tcp-port-used": "^0.1.2",
+    "watchman": "^0.1.8"
+  },
+
+  "bin": {
```

Comment:
I think you were actually correct since we only have one executable https://docs.npmjs.com/files/package.json